src/widget.js

```diff
@@ -65,11 +65,12 @@ var widget = {
         }
     },
 
+
     constructURL:(keyword, users) => {
         var url = "https://api.github.com/search/repositories?q=";
```

Comment:
This may be a problem which will be fixed during the typescript code refactor? but here's code duplication between these two js file. Can we have a helper/utitlity js to define this function to constructUrl and both scripts to call that? As of right now, if we decide to change the sort method or per page results, you will have to change two places.

Reply:
Yes, my first unit-testing PR which I have ready addresses this problem, I can move those changes into this PR, or we can push through and ready the next PR

Reply:
good to know. We can move on and fix this in the next PR